src/goGenerateTests.ts

```diff
@@ -98,8 +98,16 @@ export async function generateTestCurrentFunction(): Promise<boolean> {
 		return Promise.resolve(false);
 	}
 	let funcName = currentFunction.name;
-	if (funcName.includes('.')) {
-		funcName = funcName.split('.')[1];
+	if (funcName.includes('.')) {  // receiverType specified
+		let funcNameParts = funcName.match(/^\(\*?(.*)\)\.(.*)$/);
```

Comment:
If we are going to use a regular expression, then the `if (funcName.includes('.'))` check is redundant right?

Do we foresee the function name to include `.` in a non receiverType case?

Reply:
Hi Ramya, thanks for your fantastic work on the extension!

I removed the `if (funcName.includes('.'))` block. I don't see an other case where the function name would include `.` Do you?

Reply:
Nope, change looks good. I have pushed a commit to fix the linting errors, we are good to merge now